src/Symfony/Component/Messenger/ParallelMessageBus.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+use Amp\Parallel\Worker\ContextWorkerPool;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+use function Amp\async;
+use function Amp\Parallel\Worker\workerPool;
+
+class ParallelMessageBus implements MessageBusInterface
+{
+    public function __construct(private array $something, private readonly KernelInterface $kernel)
+    {
+    }
+
+    public static ?ContextWorkerPool $worker = null;
```

Comment:
can you move properties before functions?

Reply:
done